app/scripts/shell/dimSearchFilter.directive.js

```diff
@@ -259,6 +259,30 @@
         } else if (term.indexOf('quality:') >= 0 || term.indexOf('percentage:') >= 0) {
           filter = term.replace('quality:', '').replace('percentage:', '');
           addPredicate("quality", filter);
+        } else if (term.indexOf('rof:') >= 0) {
+          filter = term.replace('rof:', '');
+          addPredicate("rof", filter);
+        } else if (term.indexOf('impact:') >= 0) {
+          filter = term.replace('impact:', '');
+          addPredicate("impact", filter);
+        } else if (term.indexOf('range:') >= 0) {
+          filter = term.replace('range:', '');
+          addPredicate("range", filter);
+        } else if (term.indexOf('stability:') >= 0) {
+          filter = term.replace('stability:', '');
+          addPredicate("stability", filter);
+        } else if (term.indexOf('reload:') >= 0) {
+          filter = term.replace('reload:', '');
+          addPredicate("reload", filter);
+        } else if (term.indexOf('magazine:') >= 0) {
```

Comment:
I wonder - should these each be top level search types, or should it be something like "stat:magazine>20"